templates/jobs/terraform-deploy.groovy.j2

```diff
@@ -50,12 +50,12 @@ set -e
 
 # GCE
 cd gce
-terraform ${action} -var env=${environment_name} -input=false ${extraopts}
+terraform ${action} -var env=${environment_name} -input=false ${extraopts} -state=${environment_name}-terraform.tfstate
```

Comment:
Don't we want ${extraopts} to be the last parameter being passed? That way, if we somehow mess up the extra args being passed, it will still use the statefile we tell it.